assets/js/main.js

```diff
@@ -46,7 +46,8 @@ window.tagsList = [
   'zebra'
 ];
 
-window.imageHeight = 655;
+
+window.imageHeight = 650;
```

Comment:
since you had to change this `650` value in like 5 places, i think it would be best to make this a `const` that is used throughout the rest of the code, like:

```
const minimumHeight = 650;
window.imageHeight = minimumHeight;

... 

if (newHeight < minimumHeight) {
...
```

(at least i _think_ "minimumHeight" is a good name cuz it seems like that is what that is doing. but i will leave that up to you!)